tests/workflows/basic_inline_prompt_node_validation/tests/test_workflow.py

```diff
@@ -0,0 +1,37 @@
+from vellum.workflows.errors.types import WorkflowErrorCode
+
+from tests.workflows.basic_inline_prompt_node_validation.workflow import BasicInlinePromptWorkflow, WorkflowInputs
+
+
+def test_run_workflow__missing_required_input_variable():
```

Comment:
[nit] If we are adding unit tests above, then these workflow-level tests are not validating anything new. A good rule of thumb I notice is that if there are no changes to `workflows/runner.py`, then we probably don't need a workflow level test